lib/controller.js

```diff
@@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
         //load view
         lnCmsClientService.getView(endpoint, params)
           .then(function(response) {
-            if (response.data.length > 0)
+            if (response.data.length > 0) {
```

Comment:
I’m not aware of that, is working fine on staging and local with this changes, since as I mentioned before if the data returned is an array will pick the entry of that array if not will pick the object instead. 

> On Apr 4, 2016, at 7:25 PM, Adam Fenton <notifications@github.com> wrote:
> 
> In lib/controller.js <https://github.com/moxie-leean/ng-cms/pull/6#discussion_r58470987>:
> 
> > @@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
> >          //load view
> >          lnCmsClientService.getView(endpoint, params)
> >            .then(function(response) {
> > -            if (response.data.length > 0)
> > +            if (response.data.length > 0) {
> is that because of the hack which Cari put in for post? Because afaik it should normally return an object not an array.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/ng-cms/pull/6/files/bfcc4884b2934af2d9d2f11fa483402048416224#r58470987>